folium/plugins/__init__.py

```diff
@@ -19,6 +20,7 @@
 
 __all__ = [
     'MarkerCluster',
+    'FastMarkerCluster',
```

Comment:
How does this compares with our `MarkerCluster` besides speed? Can we replace the current one with this?